### PR TITLE
Change deprecated redis cache

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: ureport/polls/tests.py [hunk 1]
```diff
@@ -1111,9 +1111,9 @@ def test_delete_poll_results_counter(self):
                                   ruleset=poll_question.ruleset_uuid, date=timezone.now(),
                                   contact='contact-uuid', completed=False)
 
-        with self.settings(CACHES={'default': {'BACKEND': 'redis_cache.cache.RedisCache',
+        with self.settings(CACHES={'default': {'BACKEND': 'django_redis.cache.RedisCache',
```
Comment (rowanseymour) [inline ureport/polls/tests.py]: Why do these settings have to be overridden on a per test basis?

Comment (norkans7) [inline ureport/polls/tests.py]: The default is the Dummy cache, so we are overriding whenever we need to consider the cached value

Comment (rowanseymour) [inline ureport/polls/tests.py]: Any reason you're using dummy cache? Don't we usually just use redis db 15?

Comment (norkans7) [inline ureport/polls/tests.py]: yep, I can update and use always redis, and adjust the place we need to ignore the cache

**Comment by norkans7** [comment]
please review

**Comment by nicpottier** [comment]
Was caching not working? What's the functional change here?

**Comment by norkans7** [comment]
Nothing changed, I just founf redis_cache was deprecated and by using django_redis, @rowanseymour  catched we were repeating the cache settings unnecessary, so I went ahead and made that adjustments

**Pull request merged**